### ISSUE TITLE
SEACASAprepro find package error

### ISSUE THREAD

**Issue by gahansen** [SNLComputation/lgrtk#49]
I did a fresh "archie pull all" this morning and I am getting the following build error:

-- LGR_USE_SEACASAprepro: ON
-- SEACASAprepro_PREFIX: 
CMake Error at bob.cmake:335 (find_package):
  By not providing "FindSEACASAprepro.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "SEACASAprepro", but CMake did not find one.

  Could not find a package configuration file provided by "SEACASAprepro"
  with any of the following names:

    SEACASApreproConfig.cmake
    seacasaprepro-config.cmake

  Add the installation prefix of "SEACASAprepro" to CMAKE_PREFIX_PATH or set
  "SEACASAprepro_DIR" to a directory containing one of the above files.  If
  "SEACASAprepro" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:14 (bob_add_dependency)

-- Configuring incomplete, errors occurred!

Does anyone know what might be happening to me?

**Comment by japlews** [comment]
I’m not sure if/how/when this dependency was injected, but it seems that you do not have a Trilinos build of Aprepro, as the message states. Either build and install SEACASAprepro under Trilinos, or set “LGR_USE_SEACASAprepro” to “OFF”!

Hope that helps,
Julia

**Comment by gahansen** [comment]
Yes - I do not have a Trilinos build of Aprepro. I'll try turning this option off and see if I can complete the build as I don't think I need Aprepro. 

CC: @rwdlnk

**Comment by gahansen** [comment]
@dpzwick - do you think I'd be fine turning off aprepro in the build, or is this used in the tests or some other way that I should consider building this dependency? I guess I'm asking if this is a required or an optional dependency?

**Comment by dpzwick** [comment]
@gahansen I guess the Aprepro update never got pushed when this package was added. If you re-pull and do an ./archie update seacas, then Aprepro (which is a part of the seacas package) should be built and linked to the lgrtk package. See this commit for the fix I just pushed now: https://github.com/SNLComputation/lgrtk-build/commit/caa51fafabcfed06276e82d59bd8ecd4a69edaa3

Alternatively, it shouldn't hurt anything to turn it off if you wanted. If there are still issues just add another comment here and we can reopen the issue.

Thanks,
David
